src/ui/components/VPNConnectionStability.qml

```diff
@@ -6,7 +6,9 @@ import QtQuick 2.5
 import QtQuick.Layouts 1.14
 import Mozilla.VPN 1.0
 import "../themes/themes.js" as Theme
-import "/glean/load.js" as Glean
+
+import org.mozilla.Glean 0.15
+import generated 0.15
```

Comment:
Would be great to have a better name. Do you think is it possible?

Reply:
I agree actually, I was thinking of renaming this to 'telemetry' but as a first cut I just wanted to keep it as close as possible to the Glean example app where they call it 'generated'